src/helpers/checkDomainToken.ts

```diff
@@ -0,0 +1,11 @@
+export default function (secret: string) {
+  const regexpMessage = /^[a-zA-Z\d]{130}$/g
```

Comment:
I hope we don't change the `lenght` of this, anyway, we can combine this `regexp` into one, something like:
`^[a-zA-Z\d]{130}-[a-zA-Z\d]{14}$`

Reply:
Did it